server/render/index.js

```diff
@@ -74,8 +75,40 @@ export function render( element, key = JSON.stringify( element ) ) {
 	//todo: render an error?
 }
 
+function getCacheKey( context ) {
+	if ( ! isSectionIsomorphic( context.store.getState() ) || context.user ) {
+		return false;
+	}
+
+	if ( isEmpty( context.query ) ) {
+		return context.pathname;
+	}
+
+	let queryParams = Object.keys( context.query );
+	const cacheConfiguration = get( getIsomorphicSectionConfiguration( context.store.getState() ), 'cache', null );
+
+	if ( cacheConfiguration && cacheConfiguration.queryParams === true ) {
+		return context.canonicalPath;
```

Comment:
Verified it isn't available:
![screen shot 2017-08-14 at 14 39 00](https://user-images.githubusercontent.com/326402/29270227-66a78a40-80fe-11e7-8c6d-ea2c923dd1d5.png)